WISE-Community/WISE
# Show unit info for teacher run items

To test:
- Load `/site` and make sure clicking on a unit in the homepage library brings up the project details dialog with all of the unit info populated.
- Log in as a teacher and select the "Unit Info" link from the menu (upper right) for run listing items in the "Class Schedule" tab of the teacher homepage (`/teacher`). Make sure the project details dialog appears with all of the unit info populated.
- In the "Browse WISE Units" tab, click on units from each of the three library tabs and make sure the project details dialog appears and populates the unit info. 

Closes #1470.
